portfolio/src/main/webapp/script.js

```diff
@@ -295,10 +295,48 @@ function buildSectionEntryDescription(description) {
   }
 }
 
+/**
+ * Processes and appends a list of greetings.
+ * 
+ * @param {!Array<string>} greetings A list of greetings.
+ */
+function renderGreetings(greetings) {
+  const mainContainer = document.querySelector('main');
+  const greetingFragments = greetings.map(buildGreetingFragment);
+
+  mainContainer.prepend(...greetingFragments);
+}
+
+/**
+ * Creates and populates a greeting fragment.
+ * 
+ * @param {string} greeting The greeting to populate the fragment.
+ * 
+ * @return {!HTMLElement} The populated fragment.
+ */
+function buildGreetingFragment(greeting) {
+  const fragment = document.createElement("div");
+  fragment.append(greeting);
+
+  return fragment;
+}
+
+/**
+ * Requests a list of greetings from "/data".
```

Comment:
No need to say where it comes from, just a general comment about what the function does is enough. Being too specific makes it hard to JSdoc hard to maintain (having to change it every time you change the endpoint, e.g.)

Reply:
Gotcha, updated!